presets/4.5/rates/rubberquads_team_rates.txt

```diff
@@ -0,0 +1,89 @@
+#$ CATEGORY: RATES
+#$ STATUS: EXPERIMENTAL
+#$ KEYWORDS: rates, freestyle, race, der_V, settek, rubberquads
+#$ AUTHOR: Volker Göschl-Covrljan
+
+#$ PARSER: MARKED
+#$ DESCRIPTION: <br>
+#$ DESCRIPTION: <img src="https://www.rubberquads.com/static/rubberquads/RQ-logo_header.png" width="200px" style="margin-left: auto; margin-right: auto; display: block;"/>
+#$ DESCRIPTION:
+#$ DESCRIPTION: #Rates from Team RubberQuads
+#$ DESCRIPTION: This is what we like to rock on our sticks.
+#$ DESCRIPTION: I (Volker) am a thumber so my center rates are pretty low to give me the ability to fly smooth and precise lines 
+#$ DESCRIPTION: My race-rates are toned down at the edge (550dps) but the same at the center to allow for a seamless transition 
+#$ DESCRIPTION: between ze flippyfloppies and racing/cinematic
+#$ DESCRIPTION: Settek is flying hybrid so his rates are quite a bit higher around center-stickder_V_free
+#$ DESCRIPTION: Be aware that we use a little throttle expo to get more precition where we need (want) it most:
+#$ DESCRIPTION: my freestyle rates have it slightly over the hover-point to help with smooth, low flying and catching dives
+#$ DESCRIPTION: Setteks mid-point is suited for fast cruising and my race-rates will keep my throttle output around 70% 
+#$ DESCRIPTION: for a wider range which reflects my current skills.
+
+#$ INCLUDE: presets/4.3/rates/defaults.txt
+
+#$ OPTION_GROUP BEGIN: I want:
+    #$ OPTION BEGIN (CHECKED): Volkers freestyle rates
+        set rateprofile_name = der_V_free
+        set thr_mid = 20
+        set thr_expo = 30
+        set rates_type = ACTUAL
+        set quickrates_rc_expo = OFF
+        set roll_rc_rate = 6
+        set pitch_rc_rate = 6
+        set yaw_rc_rate = 6
+        set roll_expo = 40
+        set pitch_expo = 40
+        set yaw_expo = 40
+        set roll_srate = 70
+        set pitch_srate = 70
+        set yaw_srate = 55
+    #$ OPTION END
+    #$ OPTION BEGIN (UNCHECKED): Volkers race rates
+        set rateprofile_name = der_V_race
+        set thr_mid = 70
+        set thr_expo = 20
+        set rates_type = ACTUAL
+        set quickrates_rc_expo = OFF
+        set roll_rc_rate = 6
+        set pitch_rc_rate = 6
+        set yaw_rc_rate = 6
+        set roll_expo = 40
+        set pitch_expo = 40
+        set yaw_expo = 40
+        set roll_srate = 55
+        set pitch_srate = 55
+        set yaw_srate = 40
+    #$ OPTION END
+    #$ OPTION BEGIN (UNCHECKED): Settek rates
+        set rateprofile_name = settek
+        set thr_mid = 30
+        set thr_expo = 30
+        set rates_type = ACTUAL
+        set quickrates_rc_expo = OFF
+        set roll_rc_rate = 11
+        set pitch_rc_rate = 11
+        set yaw_rc_rate = 11
+        set roll_expo = 50
+        set pitch_expo = 50
+        set yaw_expo = 50
+        set roll_srate = 70
+        set pitch_srate = 70
+        set yaw_srate = 65
+    #$ OPTION END
+#$ OPTION_GROUP END   
+
+#$ OPTION_GROUP BEGIN: loaded rates into:
```

Comment:
i think this should be before setting rates?

Reply:
makes sense!